python/cudf/cudf/core/index.py

```diff
@@ -1563,11 +1564,39 @@ def __contains__(self, item):
         else:
             return False
 
-    def copy(self, deep=True):
+    def copy(self, deep=True, name=None, dtype=None):
```

Comment:
These kwargs need to be reordered per Pandas: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Index.copy.html

`deep` should default to `False` and adding the `names` kwarg for compatibility.
```suggestion
    def copy(self, name=None, deep=False, dtype=None, names=None):
```

Reply:
Yes, I started using `name` as the first parameter. Some dependent methods called this not by keyword but positional arguments. So what should go for `deep` goes to `name`. Is it OK to change dependent method's usage?

Reply:
Yes, that's a bug within our code then 😄